snippets/logging-5.7-outputs-snip.adoc

```diff
@@ -0,0 +1,23 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+.Logging 5.7 outputs
+[options="header"]
+|====================================================================================================
+| Output                | Protocol           | Tested with                      | Fluentd  | Vector
+| Cloudwatch            | REST over HTTP(S)  |                                  | &#10003; | &#10003;
+| Elasticsearch v6      |                    | v6.8.1                           | &#10003; | &#10003;
+| Elasticsearch v7      |                    | v7.12.2, 7.17.7                  | &#10003; | &#10003;
+| Elasticsearch v8      |                    | v8.4.3                           |          | &#10003;
```

Comment:
fluent does support forwarding to ES8 ([LOG-2544](https://issues.redhat.com/browse/LOG-2544))